modules/cas_cache/layer_cache_management.c

```diff
@@ -437,15 +475,37 @@ int cache_mngt_prepare_core_cfg(struct ocf_mngt_core_config *cfg,
 		struct kcas_insert_core *cmd_info)
 {
 	struct block_device *bdev;
+	static char core_name[OCF_CORE_NAME_SIZE];
```

Comment:
static array again, potential races if called in parallel.

Reply:
Done.